lxd/daemon_config.go

```diff
@@ -45,4 +45,8 @@ func daemonConfigSetProxy(d *Daemon, config *clusterConfig.Config) {
 		config.ProxyHTTP(),
 		config.ProxyIgnoreHosts(),
 	)
+
+	if d.oidcVerifier != nil {
```

Comment:
The OIDC config automatically refreshes after the default refresh interval, but if `core.http_proxy` is set in-between refreshes the OIDC verifier will still be using the old HTTP client and it will fail.

Reply:
No, the HTTP client is set when the relying party is configured and there is no way to set it afterwards.